src/miscsetup.jl

```diff
@@ -0,0 +1,8 @@
+using Images, Kalman, GaussianDistributions
+
+import Base.LinAlg.normalize!
+function normalize!(x::AbstractArray)
+    l, u = extrema(x)
+    (x - l)/(u - l)
+end
+greyscale(A) = map(x->RGB(x,x,x), normalize!(copy(A)));
```

Comment:
Is the purpose of these function just for the presentation or is it *actually* important for the Kalman filter to get values that are between 0 and 1?

Reply:
No, the filter does not see the image, it *only* sees it through the calls of `track`.

Reply:
OK, so there's no real need for them in the sense of the filter.